bridge/image.yaml

```diff
@@ -3,7 +3,7 @@ schema_version: 1
 name: amq7/amq-streams-bridge-rhel7
 description: "AMQ Streams image for running the Apache Kafka bridge"
 version: "1.6.0"
-from: rh-osbs/ubi7-minimal:7-released
+from: registry.redhat.io/ubi7/ubi-minimal
```

Comment:
do we need the entire registry path? or just `ubi7/ubi-minimal` is enough? Without specifying a tag I guess it's going to use `latest`, is it the right one?

Reply:
Unfortunately, we need the full path to the public registry, but the good thing about this is that it guarantees we will only spin on released images now that the "-released" tag is being removed from the internal and external registries. Yes, this will default to "latest", so we will pick up the latest CVE patches, from what I understand from the deprecation docs (which I will send you offline), this will be equivalent to the "7-released" tag